paddle/fluid/operators/detail/grpc_server.cc

```diff
@@ -30,9 +30,13 @@ enum CallStatus { PROCESS = 0, FINISH };
 class RequestBase {
  public:
   explicit RequestBase(GrpcService::AsyncService* service,
-                       ::grpc::ServerCompletionQueue* cq,
+                       ::grpc::ServerCompletionQueue* cq, bool sync_mode,
```

Comment:
maybe `is_sync` or just `sync` can tell the meaning?

Reply:
I think `sync_mode` means it works in a mode, but `is_sync` means itself is async. So I think `sync_mode` is better.